packages/react/src/themes/teams/components/Icon/svg/ProcessedIcons/icons-msft-stream.tsx

```diff
@@ -10,4 +10,5 @@ export default {
     </svg>
   ),
   styles: {},
+  exportedAs: 'msft-stream',
```

Comment:
```suggestion
  exportedAs: 'microsoft-stream',
```

Can we use full name?